### ISSUE TITLE
Disable Auto Import? (Clangd)

### ISSUE THREAD

**Issue by arowe92** [hrsh7th/nvim-compe#508]
I Cannot figure out if there is a way to disable the auto import feature.

#### Is your feature request related to a problem? Please describe.
Almost every time I auto complete to a class in c++ I end up having to go delete the auto-import. 
It auto imports on my .cpp files which for my project should only have the single import for their corresponding header. 
It also imports even when I already have visibility to the function from a imported header. 
Its definitely a nuisance to me and can't figure out how to disable it. 
Basically the imports are all figured out for my code most of the time and rarely modify them. 

#### Describe the solution you'd like
To be able to disable Auto import

**Comment by SmiteshP** [comment]
Have you tried launching clangd with "--header-insertion=never" flag?
While calling setup for clangd you can pass it like [this](https://github.com/SmiteshP/Dotfiles/blob/98f3baa644b3db9dd89da6519ee48523ce9f27eb/.config/nvim/lua/config/lsp/server/clangd.lua#L4).

**Comment by hrsh7th** [comment]
Hm... currently, we can't disable auto-importing via nvim-compe itself.

@SmiteshP Thank you!

@arowe92 Probably, you can use it :bow:
